pkg/apis/gpte/v1alpha1/nexus_types.go

```diff
@@ -0,0 +1,54 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// NexusSpec defines the desired state of Nexus
+// +k8s:openapi-gen=true
+type NexusSpec struct {
```

Comment:
do these types already exist? Which operator are we using to do the install?

Reply:
No, it's an Ansible operator: https://github.com/wkulhanek/nexus-operator